R/zchunk_L211.ag_nonco2.R

```diff
@@ -1,27 +1,26 @@
 #' module_emissions_L211.ag_nonco2
 #'
-#' Briefly describe what this chunk does.
+#' Processes agriculture, agricultural waste burning, and animal emissions.
+#' Writes out emissions coefficients for biomass and ag waste burning and non-GHG max reductions and steepness.
```

Comment:
These two lines are not very specific. It would good to say (here or below in `@details`) exactly what an emissions coefficient is, what steepness means, etc.